.pre-commit-config.yaml

```diff
@@ -8,44 +8,56 @@
 # - Run on all files:   pre-commit run --all-files
 # - Register git hooks: pre-commit install --install-hooks
 #
+ci:
+  autoupdate_interval: monthly
 repos:
   # Autoformat: Python code, syntax patterns are modernized
   - repo: https://github.com/asottile/pyupgrade
     rev: v3.3.1
     hooks:
       - id: pyupgrade
         args:
-          - --py38-plus
+          - --py37-plus
```

Comment:
This project is at py38+

https://github.com/jupyterhub/docker-image-cleaner/blob/main/setup.py#L18